data/basics/userBalance.go

```diff
@@ -420,6 +421,14 @@ type AssetParams struct {
 	// Clawback specifies an account that is allowed to take units
 	// of this asset from any account.
 	Clawback Address `codec:"c"`
+
+	// GlobalFrozen specifies whether the asset is frozen across
+	// all accounts.
+	GlobalFrozen bool `codec:"gf"`
+
+	// LastAssetFreeze specifies the last time the asset was frozen
+	// or unfrozen globally.
+	LastAssetFreeze uint64 `codec:"l"`
```

Comment:
So the simplification I'm recommending is:
```suggestion
	// LastGlobalFreeze specifies the last time (as a transaction counter) the asset was frozen
	// globally, or 0 if the last global change was unfreeze.
	LastGlobalFreeze uint64 `codec:"l"`
```

Reply:
Updated to this wording, although similarly to the asset holding I simply used `l` as the codec. Having them both as `l` may be confusing so happy to change them, but then again there's `f` for asset holding Frozen and `f` for asset param Freeze address.

Reply:
Again, my feeling is that you want to "name" it with the noun, not the adjective without the noun.

Reply:
I updated to `fc` for holding `LastFreezeChange` and `gf` for params `LastGlobalFreeze`.